### PR TITLE
Example updates

### PR DESCRIPTION
**Pull request by l50**
# Proposed Changes

This pull request (PR) proposes documentation updates for the TTPForge project based on the changes made in the recent commit. The changes include updates to several README files within the `ttps/examples` directory, specifically:

1. Updates to `args` example:
   - Created documentation to provide a description of the TTP's purpose and steps.
   - Included expected output for each step of the TTP.

2. Created `mitre` example
   - Created a new `mitre` example with a README describing the functionality of the step and how to run it.
   - Included expected output for each step of the TTP.

4. Updates to `outputs` example:
   - Created documentation to provide a description of the TTP's purpose and steps.
   - Included expected output for each step of the TTP.

5. Updates to `edit-step` example
   - Created documentation to provide a description of the TTP's purpose and steps.
   - Included expected output for each step of the TTP.

6. Updates to `sub-ttps` example
   - Created documentation to provide a description of the TTP's purpose and steps.
   - Included expected output for each step of the TTP.

7. Created `fetchuri` example:
   - Created a new `fetchuri` example with a README describing the functionality of the step and how to run it.
   - Included expected output for each step of the TTP.

## Related Issue(s)

No related issues are associated with this PR.

## Testing

N/A

## Documentation

Updated and added documentation for various TTP examples within the `ttps/examples` directory.

## Screenshots/GIFs (optional)

N/A

## Checklist

- [x] Ran `mage runprecommit` locally and fixed any issues that arose.
- [x] Curated your commit(s) so they are legible and easy to read and understand.
- [x] 🚀

### CODE REVIEW

**Comment by d3sch41n** [comment]
Back from PTO - I need to review the targets TTPForge PR today, however everything else looks fine so if you want to set the targets docs to the side for a bit, can merge everything else now - LMK

**Review by d3sch41n — CHANGES_REQUESTED** [review]
Plz remove `ignore-errors`

**Comment by l50** [comment]
> Plz remove `ignore-errors`

Whoops, good catch! Done.

File: ttps/examples/steps/edit-step/edit-step.yaml [hunk 1]
```diff
@@ -6,10 +6,11 @@ steps:
     inline: |
       set -e
 
+      echo -e "Target file pre-edit:"
       cat file-to-edit.txt
   - name: edit-target-file
     edit_file: "file-to-edit.txt"
-    backup_file: "my-backup.txt"
+    backup_file: "/tmp/my-backup.txt"
```
Comment (d3sch41n) [inline ttps/examples/steps/edit-step/edit-step.yaml:13]: I'll fix this first

Comment (l50) [reply ttps/examples/steps/edit-step/edit-step.yaml:13]: Sounds good. Yeah, basically it drops the backup in the CWD, which is different from where Forge expects it to be. Hence the absolute path.

**Review by d3sch41n — APPROVED** [review]
i'll go back and update the edit for the appropriate fix later

**Pull request merged**